HW03_mcfarlands.py

```diff
@@ -0,0 +1,127 @@
+#################################################################################
+# Author: Samuel McFarland
+# Username: mcfarlands
+#
+# Assignment: Homework 03
+# Purpose: Use turtles to draw a complex picture, get acquainted with functions, and learn to use Git pull requests.
+# Google Doc Link: https://docs.google.com/document/d/1Xg2nZ8jbQbaRZrFfuFfSr1sExkCC4W9jZvHoHLHrQH4/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: https://rgbcolorcode.com/ - Used for finding RGB values used in the code.
+#
+#
+#################################################################################
+
+import turtle   # Allows us to use the turtle module in the code.
+
+
+def draw_grass(t):  # Function that draws grass in the window.
```

Comment:
Did you want the grass centered in the screen? It is skewed to the right for me.